sherpa/stats/__init__.py

```diff
@@ -681,6 +703,14 @@ class Chi2DataVar(Chi2):
     in the fit. The background term appears only if an estimate of the
     background has been subtracted from the data.
 
+    A(B) is the off-source "area", which could be
+    the size of the region from which the background is extracted, or
+    the length of a background time segment, or a product of the two,
+    etc.; and A(S) is the on-source "area". These terms may be defined
+    for a particular type of data: for example, PHA data sets A(B) to
+    `BACKSCAL * EXPOSURE` from the background data set and A(S) to
+    `BACKSCAL * EXPOSURE` from the source data set.
```

Comment:
Technically we also have to deal with AREASCAL but I'm not sure that's worth mentioning here (as it is a bit odd).